### PR TITLE
Some copy edits

### PR DESCRIPTION
**Pull request by RacheltheEditor**

### CODE REVIEW

**Comment by RacheltheEditor** [comment]
@jlevy it says there's one conflict but nothing is showing up in Atom 😞

File: README.md [hunk 1]
```diff
@@ -644,7 +644,7 @@ A startup goes through [several stages of growth](#stages-of-a-startup) as it ra
 based on the hope and expectation that the company will grow and make more money in the
 future.
 
-🄳 Companies add (or “issue”) shares during fundraising, which can then be awarded to
+🄳 Companies add (or “issue”) shares during fundraising, which can then be traded to
```
Comment (jlevy) [inline README.md]: Revert on this PR.

"Traded" has wrong connotations (e.g. trading floors, or that there is a broker). Brad used "exchange," which is more accurate. But making that requires additional rewording, probably "can be exchanged for cash from investors". But regardless, that change is not a copy edit and needs to be a different PR.

File: README.md [hunk 2]
```diff
@@ -1329,8 +1329,8 @@ Key considerations include:
 - Another path is to
   [split the difference](http://stockoptioncounsel.com/blog/early-expiration-of-startup-stock-options-part-2-the-full-10-year-term-solution/2017/3/28)
   and give extended windows only to longer-term employees.
-- Taken together, it’s clear many employees were not clear on the nuances of this before
-  joining companies in the past, and some
+- Taken together, it’s evident many employees have not always been clear on the nuances of
```
Comment (jlevy) [inline README.md]: Rewording "clear" is a good copy edit.

But remove "always".

Also (unrelated but while editing this): "before joining" -> "when joining"

File: README.md [hunk 3]
```diff
@@ -1934,7 +1934,7 @@ It also caused Congress to grant a one-time forgiveness, the odds of which happe
 are very low.
 
 🄳 The catastrophic scenario where exercising ISOs triggers a large AMT bill, with no
-ability to sell the stock to pay taxes, is called the **AMT trap**. This
+ability to sell the stock to pay taxes, is sometimes called the **AMT trap**. This
```
Comment (jlevy) [inline README.md]: 👍

File: README.md (hunk 2 above)
Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (RacheltheEditor) [reply README.md]: K. Reverting for now and will issue a new PR.

File: README.md [hunk 4]
```diff
@@ -26,16 +26,18 @@ Decisions related to [negotiating an offer](#offers-and-negotiations) and
...
-[catastrophic tax pitfalls](#tax-dangers) because they didn’t anticipate the tax consequences.
+[catastrophic tax pitfalls](#tax-dangers) because they didn’t anticipate the tax consequences
+of their decisions.
 
 Understanding the technicalities of equity compensation does not guarantee that fortune
 will smile upon you as warmly as it did [the early hires](#history-and-significance) of
-Facebook. But a thorough overview can help you be informed when
-[discussing with professionals](#seeking-professional-advice) who may help you, make better
-decisions, avoid some of the most [common](#tax-dangers) and [costly](#the-amt-trap) mistakes.
+Facebook. But a thorough overview can help you avoid some [common](#tax-dangers) and
```
[the remark this answers is not in the document]
Comment (RacheltheEditor) [reply README.md]: This is a copyedit because the list had syntactical and rhythmic issues and an order of operations was not implied by the original list. If you want it to read that we want everyone to talk to a professional after reading this Guide, that's a different kind of construction. ("A thorough overview can prepare you to meet with professionals who can help you make better decisions and avoid common and costly mistakes." But we also want to say that the wisdom in this Guide will help you avoid mistakes and make decisions, too, so, the list doesn't need to have an order.) Reverted w/ added conjunction.

**Comment by jlevy** [comment]
Looks good. Thanks!
